test/checks/missing_enable_comment_test.rb

```diff
@@ -90,4 +90,151 @@ def test_specific_checks_disabled_and_reenabled_with_all_checks_disabled
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+        {% assign x = 1 %}
+      END
+    )
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_adds_missing_enable_after_last_check_when_multiple_checks_disabled
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable OneCheck, TwoCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-enable{% endcomment %}
+      END
+    }
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable OneCheck, TwoCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable TracerCheck, AnotherCheck{% endcomment %}
+          {% assign x = 1 %}
+        {% comment %}theme-check-disable{% endcomment %}
+          {% assign x = 1 %}
+      END
+    )
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_comment_insert_location
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+        {% comment %}theme-check-enable{% endcomment %}
+        {% assign x = 1 %}
+      END
+    }
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+        {% assign x = 1 %}
+      END
+    )
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_does_not_correct_first_line_disable
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+      END
+    }
+
+    sources = fix_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        {% comment %}theme-check-disable{% endcomment %}
+        {% assign x = 1 %}
+      END
+    )
+
+    sources.each do |path, source|
+      assert_equal(expected_sources[path], source)
+    end
+  end
+
+  def test_html_check
+    expected_sources = {
+      "templates/index.liquid" => <<~END,
+        <p>Hello, world</p>
+        {% comment %}theme-check-disable ImgWidthAndHeight{% endcomment %}
+        <img src="a.jpg">
+        {% comment %}theme-check-enable ImgWidthAndHeight{% endcomment %}
```

Comment:
I'm not sure the implementation works.
- What if the comment was added a couple of lines up?
- What if there are multiple offenses? e.g. two lines with `<img src="a.jpg">`?
- What if the comment is on the same line of the offense?
```
{% theme-check-disable ImgWidthAndHeight %}<img src="a">
<img src="a">
```

Reply:
The current implementation adds the closing comment one line after the disable comment regardless of how many offenses there are.